docroot/themes/custom/uids_base/templates/uids/blockquote.html.twig

```diff
@@ -0,0 +1,24 @@
+{#
+ # Renders a <blockquote> with footer, citation and attributes if available.
+ #
+ # Available properties of the blockquote object:
+ # * blockquote_attributes
+ # * blockquote_cite
+ # * blockquote_cite_url
+ # * blockquote_footer
+ # * blockquote_content
+ #
+ #}
+
+{% if (blockquote_cite_url is not empty) %}
+  <blockquote {{ blockquote_attributes }} cite="{{ blockquote_cite_url }}">
+{% else %}
+  <blockquote {{ blockquote_attributes }}>
+{% endif %}
+    {{ blockquote_content }}
+    {% if blockquote_footer is not empty or (blockquote_cite is not empty) %}
```

Comment:
I think this `or` condition should be changed to `and`. Otherwise, I think the `elseif` will never run.